src/Illuminate/Http/Client/Factory.php

```diff
@@ -191,6 +191,28 @@ public function assertSent($callback)
         );
     }
 
+    /**
+     * Assert that a request / response pair was not recorded matching a given truth test.
+     *
+     * @param  callable  $callback
+     * @return void
+     */
+    public function assertNotSent($callback)
+    {
+        PHPUnit::assertFalse(
+            $this->recorded($callback)->count() > 0,
+            'An expected request was recorded.'
```

Comment:
You meant "Unexpected request was recorded." right?

Reply:
Yep 👍